react/features/analytics/functions.js

```diff
@@ -161,13 +179,8 @@ function _loadHandlers(scriptURLs = [], handlerConstructorOptions) {
         // check the old location to provide legacy support
         const analyticsHandlers = [
             ...getJitsiMeetGlobalNS().analyticsHandlers,
-            ...window.analyticsHandlers,
-
-            // NOTE: when we add second handler it will be good to put all
```

Comment:
Please move this note to where you initialize Amplitude now, so we can remember.

Reply:
I don't think this is valid anymore because the initialization of the build-in analytics is separate. This is because we need to initialize Amplitude before SET_ROOM is executed.

Reply:
I think it still is. We are currently not using the built-in Google Analytics handler, are we?

Reply:
it's implemented now